docs/create_own_rule.md

```diff
@@ -32,7 +32,7 @@ final class MyFirstRector extends AbstractRector
     public function getNodeTypes(): array
     {
         // what node types are we looking for?
-        // pick any node from https://github.com/rectorphp/rector/blob/master/docs/nodes_overview.md
+        // pick any node from https://github.com/rectorphp/php-parser-nodes-docs/blob/master/docs/nodes_overview.md
```

Comment:
At least this is the url used in the rector readme

Reply:
What about https://github.com/rectorphp/php-parser-nodes-docs/?

Reply:
Thats what this PR achieves. In the readme the url https://github.com/rectorphp/php-parser-nodes-docs/blob/master/docs/nodes_overview.md Is already used

Reply:
I mean direct repository link: https://github.com/rectorphp/php-parser-nodes-docs/
The docs is in `README.md` now too as less surprising location.

The old link is just for BC :)

Reply:
Adjusted per feedback